cxx-sensors/src/main/java/org/sonar/cxx/sensors/visitors/CxxFileLinesVisitor.java

```diff
@@ -227,22 +253,25 @@ public void leaveFile(AstNode astNode) {
     if (LOG.isDebugEnabled()) {
       LOG.debug("CxxFileLinesVisitor: '{}'", inputFile.uri().getPath());
       LOG.debug("   lines:           '{}'", inputFile.lines());
-      LOG.debug("   executableLines: '{}'", EXECUTABLE_LINES);
-      LOG.debug("   linesOfCode:     '{}'", LINES_OF_CODE);
-      LOG.debug("   linesOfComments: '{}'", LINES_OF_COMMENTS);
+      LOG.debug("   executableLines: '{}'", Sets.newHashSet(executableLines));
+      LOG.debug("   linesOfCode:     '{}'", Sets.newHashSet(linesOfCode));
+      LOG.debug("   linesOfComments: '{}'", Sets.newHashSet(linesOfComments));
     }
   }
```

Comment:
Wondering if it would not be better to clear lists also here to safe memory. Don't know how long the visitor does life?

Reply:
The lifetime of these containers was extended artificially in order to ease testing. I've rewrite tests instead